pages/create.tsx

```diff
@@ -1,9 +1,5 @@
+import MarketEditor from "components/create/editor/Editor";
 import { NextPage } from "next";
-import dynamic from "next/dynamic";
-
-const MarketEditor = dynamic(() => import("components/create/editor/Editor"), {
```

Comment:
@yornaath do you remember this being needed for any reason? Removing it makes loading quite a bit faster